src/scheduledUpdater.js

```diff
@@ -0,0 +1,55 @@
+require('dotenv').config();
+const jwt = require('jsonwebtoken');
+const requestPromise = require('request-promise');
+const sgMail = require('@sendgrid/mail');
+const dependencies = require('./schedulerDependencies');
+
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+const decoded = jwt.decode(process.env.OAUTH_ACCESS_TOKEN);
+const { exp } = decoded;
+const currentDate = new Date().getTime() / 1000;
+const difference = exp - currentDate;
+const daysRemaining = Math.round(difference / (24 * 60 * 60));
+
+/**
+ * @description Gets the auth0 token and sends an update request to heroku
+ *
+ * @param {number} days take number of days remaining for the current token
+ * @returns {void}
+ */
+const getToken = async (days) => {
+  let accessToken = null;
+  const daysBeforeExpiration = process.env.DAYS_TO_TOKEN_EXPIRATION;
+  if (days < daysBeforeExpiration) {
+    try {
+      const options = {
+        method: 'POST',
+        url: 'https://steps.auth0.com/oauth/token',
+        headers: { 'content-type': 'application/json' },
+        body: `{
+          "client_id":"${process.env.AUTH0_CLIENT_ID}",
+          "client_secret":"${process.env.AUTH0_CLIENT_SECRET}",
+          "audience":"http://steps-admin.herokuapp.com",
+          "grant_type":"client_credentials"
+        }`
```

Comment:
I think the body should be:
```
{
    "client_id":`${process.env.AUTH0_CLIENT_ID}`,
    "client_secret":`${process.env.AUTH0_CLIENT_SECRET}`,
    "audience":`http://steps-admin.herokuapp.com`,
    "grant_type":`client_credentials`
}
```
instead of 
```
`{
     "client_id":"${process.env.AUTH0_CLIENT_ID}",
     "client_secret":"${process.env.AUTH0_CLIENT_SECRET}",
     "audience":"http://steps-admin.herokuapp.com",
     "grant_type":"client_credentials"
 }`
```
What do you think @lukorito ?

Reply:
I tried that @meshack-mbuvi. It returns an error with `invalid JSON object`

Reply:
This method is also appropriate @meshack-mbuvi. @lukorito, something else you can do is to put the body in a separate object like so:
```
const body = {
          "client_id":"${process.env.AUTH0_CLIENT_ID}",
          "client_secret":"${process.env.AUTH0_CLIENT_SECRET}",
          "audience":"http://steps-admin.herokuapp.com",
          "grant_type":"client_credentials"
        }
```
and then line 30 becomes `body: JSON.stringify(body)`. That should work.

Reply:
What about this 
```
{
    "client_id":process.env.AUTH0_CLIENT_ID,
    "client_secret":process.env.AUTH0_CLIENT_SECRET,
    "audience":"http://steps-admin.herokuapp.com",
    "grant_type":"client_credentials"
}
```

Reply:
@meshack-mbuvi wont work

Reply:
Thanks @andela-kakpobome . @PriscillaSam's comment worked